src/org/labkey/test/tests/CreateProjectCommand.java

```diff
@@ -0,0 +1,93 @@
+package org.labkey.test.tests;
```

Comment:
Could you put this with the other test commands?
```suggestion
package org.labkey.remoteapi.admin;
```

Reply:
Sure